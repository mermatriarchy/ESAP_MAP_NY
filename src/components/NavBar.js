import React from 'react';

export default function NavBar() {
  return (
    <nav>
      Nav goes up here.
    </nav>
  );
}
