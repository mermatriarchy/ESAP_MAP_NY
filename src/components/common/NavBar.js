import React from "react";
import { Nav, Navbar, Container} from "react-bootstrap";

export default function NavBar() {
  return (
    <Navbar bg="dark" variant="dark" expand="lg">
      <Container>
        <Navbar.Brand href="/">ESAP Needle Map</Navbar.Brand>
        <Navbar.Toggle aria-controls="basic-navbar-nav" />
        <Navbar.Collapse className="justify-content-end">
          <Nav>
            <Nav.Link href="/">Map</Nav.Link>
            <Nav.Link href="/about">About</Nav.Link>
          </Nav>
        </Navbar.Collapse>
      </Container>
    </Navbar>
  );
}
