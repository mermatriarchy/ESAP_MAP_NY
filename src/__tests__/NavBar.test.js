import React from 'react';
import renderer from 'react-test-renderer';

import NavBar from '../components/common/NavBar';

it('correctly renders About component', () => {  
    const NavBarComponent = renderer.create(<NavBar />).toJSON();
    expect(NavBarComponent).toMatchSnapshot();
});