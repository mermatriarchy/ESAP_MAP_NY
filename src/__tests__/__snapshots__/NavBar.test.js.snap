// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders About component 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <a
    className="navbar-brand"
    href="#home"
  >
    ESAP Needle Map
  </a>
  <button
    aria-controls="basic-navbar-nav"
    aria-label="Toggle navigation"
    className="navbar-toggler collapsed"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    aria-expanded={null}
    className="justify-content-end navbar-collapse collapse"
  >
    <div
      className="navbar-nav"
      onKeyDown={[Function]}
    >
      <a
        className="nav-link"
        data-rb-event-key="/"
        href="/"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        Map
      </a>
      <a
        className="nav-link"
        data-rb-event-key="/about"
        href="/about"
        onClick={[Function]}
        onKeyDown={[Function]}
      >
        About
      </a>
    </div>
  </div>
</nav>
`;
